composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.3-dev"
+            "dev-master": "0.4-dev"
```

Comment:
Do you want to align the versions between plugins and the main repo? I don't think it is a good idea. (In the case this is a mistake: 0.3 is the next IIRC)